tests/test_model.py

```diff
@@ -1501,6 +1461,96 @@ def test_add_boundary_condition_by_geometry_ids(self, create_default_3d_soil_mat
 
             TestUtils.assert_almost_equal_geometries(expected_geometry, model_part.geometry)
 
+    def test_add_load_by_geometry_ids(self, create_default_3d_soil_material: SoilMaterial,
+                                      create_default_point_load_parameters: PointLoad,
+                                      create_default_line_load_parameters: LineLoad,
+                                      create_default_surface_load_parameters: SurfaceLoad):
+        """
+        Test if a load is added correctly to the model. Here the load is added to the model by
+        specifying the geometry ids to which the load should be applied.
+
+        Args:
+            - create_default_3d_soil_material (:class:`stem.soil_material.SoilMaterial`): A default soil material.
+            - create_default_point_load_parameters (:class:`stem.load.PointLoad`): default point load parameters.
+            - create_default_moving_load_parameters (:class:`stem.load.MovingLoad`): default moving load parameters.
```

Comment:
missing surface load in docstring. Also create_default_moving_load_parameters  is not present in the input